### PR TITLE
Adding additional operations

### PR DESCRIPTION
**Pull request by SparshithNR**
1. Added rmdirSync with recursive:true
2. Added unlinkSync, symlinkOrCopySync
3. Added tests for the same
4. Added README changes

### CODE REVIEW

**Comment by SparshithNR** [comment]
#6

File: test/unit-test.js [hunk 1]
```diff
@@ -34,10 +35,36 @@ describe('output-wrapper', function() {
...
+    output.mkdirSync('test');
+    expect(fs.existsSync(`${temp.name}/test`)).to.be.true;
+    output.writeFileSync('test.md', 'test');
+    output.rmdirSync('./', {
+      recursive: true
+    });
+    expect(fs.existsSync(`${temp.name}/test`)).to.be.false;
+  });
+
+  it('can remove folder non recursive', function() {
```
Comment (stefanpenner) [inline test/unit-test.js]: Can we test that non recursive fails if it is used to delete a directory which contains content? And that it fails the same way the new version fails?

Comment (SparshithNR) [reply test/unit-test.js]: https://github.com/SparshithNR/broccoli-output-wrapper/blob/61e46ea7c427dff35b1c5c43cf32408cef209cd3/test/unit-test.js#L55

I have that test added.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "broccoli-output-wrapper",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
[the remark this answers is not in the document]
Comment (SparshithNR) [reply package.json]: Bumped to `3.1.0`

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "broccoli-output-wrapper",
-  "version": "3.0.0",
+  "version": "3.1.0",
```
Comment (rwjblue) [inline package.json]: FWIW, it's generally better to leave the version number until you are ready to actually publish. That's my process at least...

Comment (SparshithNR) [reply package.json]: I should also follow that too. I am simply bumping versions that I don't like. I will make a note of this.

**Pull request merged**